postgres_base_block.py

```diff
@@ -1,14 +1,96 @@
+import re
+from psycopg2 import connect
+
 from nio.block.base import Block
-from nio.util.discovery import discoverable
-from nio.properties import VersionProperty
+from nio.command import command
+from nio.properties import (VersionProperty, StringProperty, IntProperty,
+                            BoolProperty, PropertyHolder, ObjectProperty)
+from nio.util.discovery import not_discoverable
+
+
+class AuthCreds(PropertyHolder):
+    username = StringProperty(title="Username", default="", allow_none=True)
+    password = StringProperty(title="Password", default="", allow_none=True)
+
+
+@not_discoverable
+@command('Connection status', method='connection_status')
```

Comment:
As a command name this is kind of weird with a space. I think that means it would have to be called with `/services/ServiceName/PostgresBlock/Connection status`. Maybe just `connection_status`? Lowercase and snake case would be more consistent with our other commands I think. Also applies to `Reconnect` and `Disconnect`